app/src/main/res/layout/fragment_supervision_response.xml

```diff
@@ -0,0 +1,144 @@
+            android:layout_height="wrap_content"
+            android:text="@string/group_name" />
+
+        <TextView
+            android:id="@+id/text_view_group_name"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="16dp"
+            android:textStyle="bold" />
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/group_supervisor" />
+
+        <TextView
+            android:id="@+id/text_view_group_supervisor_name"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="16dp"
+            android:textStyle="bold" />
+
+
+    </LinearLayout>
+
+    <View
+        android:id="@+id/view_divider_below_schedule"
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_below="@+id/linear_layout_schedule_view"
+        android:background="@color/colorAccent" />
+
+    <LinearLayout
+        android:id="@+id/linear_layout_response"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/view_divider_below_schedule"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentEnd="true"
+        android:layout_margin="20dp"
+        android:layout_marginStart="20dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginEnd="20dp"
+        android:layout_marginBottom="20dp"
+        android:orientation="vertical"
+        android:textColor="@color/black"
+        android:textSize="36sp"
+        android:visibility="invisible">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal">
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/you_last" />
+
+            <TextView
+                android:id="@+id/text_view_schedule_user_response_text"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="4dp"
+                android:layout_marginEnd="4dp"
+                android:textStyle="bold" />
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/this_schedule_on" />
+
+        </LinearLayout>
+
+        <TextView
+            android:id="@+id/text_view_schedule_user_response_time"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:textStyle="bold" />
+
+    </LinearLayout>
```

Comment:
I think there's some leftover/unused xml in here, or xml with bad ids, because there are references to `schedule`. Please lean up this xml file.

I'm just highlighting this section, but you should check the whole xml

Reply:
Yes the second LInearLayout with id "linear_layout_response" is useless